modules/navier_stokes/src/tests/WedgeFunction.C

```diff
@@ -13,11 +13,12 @@ InputParameters validParams<WedgeFunction>()
   InputParameters params = validParams<Function>();
   params.addRequiredParam<Real>("alpha_degrees", "The wedge half-angle size (in degrees) used in computing 'f' below.");
   params.addRequiredParam<Real>("Re", "The Reynolds number used in computing 'f' below.");
-  params.addRequiredParam<unsigned int>("component", "The component (x==0, y==1) of the velocity we are computing.");
+  params.addRequiredRangeCheckedParam<unsigned int>("var_num", "var_num<3", "The variable (0=vel_x, 1==vel_y, 2==p) we are computing the exact solution for.");
```

Comment:
I thought about that, but this is only for a test, and I also use it to index into a vector, so I think it makes more sense as an int.